app/services/safe_downloader.rb

```diff
@@ -48,6 +48,7 @@ def encode_url(url)
     def patch_if_less_than_10k(downloaded_file, tempfile)
       return downloaded_file unless downloaded_file.is_a?(StringIO)
 
+      tempfile = Tempfile.new
```

Comment:
Don't you receive `tempfile` as an argument in this method, do you?

Reply:
Wow, yeah - let me redo this PR.